anthos-bm-gcp-terraform/main.tf

```diff
@@ -108,6 +108,33 @@ module "create_service_accounts" {
 }
 
 module "instance_template" {
+  source  = "terraform-google-modules/vm/google//modules/instance_template"
+  version = "~> 8.0"
+  depends_on = [
+    module.enable_google_apis_primary,
+    module.enable_google_apis_secondary
+  ]
+  # fetched from previous module to explicitely express dependency
+  project_id           = module.enable_google_apis_secondary.project_id
+  region               = var.region           # --zone=${ZONE}
+  source_image         = var.image            # --image=ubuntu-2004-focal-v20210429
+  source_image_family  = var.image_family     # --image-family=ubuntu-2004-lts
+  source_image_project = var.image_project    # --image-project=ubuntu-os-cloud
+  machine_type         = var.machine_type     # --machine-type $MACHINE_TYPE
+  disk_size_gb         = var.boot_disk_size   # --boot-disk-size 200G
+  disk_type            = var.boot_disk_type   # --boot-disk-type pd-ssd
+  network              = var.network          # --network default
+  tags                 = var.tags             # --tags http-server,https-server
+  min_cpu_platform     = var.min_cpu_platform # --min-cpu-platform "Intel Haswell"
+  can_ip_forward       = true                 # --can-ip-forward
+  # Disable oslogin explicitly since we rely on metadata based ssh-key (issues/70).
```

Comment:
**Question:** Is the exclusion of `enable_nested_virtualization = var.enable_nested_virtualization` intentional?

Reply:
Yes - As this template has been re-scoped to just the control plane and admin workstation, there is no need for nested virtualization.  Nested virtualization was added to enable the VM Runtime feature, and is only needed on worker nodes.